Makefile

```diff
@@ -118,9 +120,16 @@ update_base_url: \
 test: all
 	pytest src emsdk/tests packages/*/test* pyodide-build -v
 
+copy-fonts:
+	mkdir -p build/fonts
+
+copy-reference-images:
+	mkdir -p build/matplotlib-test
+	cp -n packages/matplotlib/reference-images/*.png build/matplotlib-test/
```

Comment:
We don't want to copy the test images into the `build` folder because then they will be distributed as part of Pyodide releases. You should make a symlink as part of the test and then remove it when the test is done. 

See here:
https://github.com/pyodide/pyodide/blob/8d7659b1e11ef602a1492b637edc557587c67e00/packages/micropip/test_micropip.py#L149

Reply:
Thank you!